### PR TITLE
PEN-1745 Primary and Secondary Font Components

### PR DESCRIPTION
**Pull request by JackHowa**
## Description

Utilising styled components to reduce the duplication of theme styles

Updated the headline block and numbered list block to show how it can be used.

## Testing

1. Run fusion with `npx fusion start -f -l @wpmedia/numbered-list-block,@wpmedia/headline-block,@wpmedia/shared-styles`
2. Goto an article page - http://localhost/arts/2020/01/17/a-huge-loss-for-lovers-of-romantic-music-chamin-correa-legendary-mexican-guitarist-dies-at-90/?_website=the-gazette
3. Verify article heading is using correct font
4. Goto an article page on a different site in the multi site set up - http://localhost/arts/2020/01/17/a-huge-loss-for-lovers-of-romantic-music-chamin-correa-legendary-mexican-guitarist-dies-at-90/?_website=the-sun
5. Verify article heading is used correct font for site
6. Also verify numbered-list-block 

## Jira Ticket
- [PEN-1745](https://arcpublishing.atlassian.net/browse/PEN-1745)

## Before

resuable-primary-font-block-before

## After

resuable-primary-font-block-after

Screen Shot 2021-02-03 at 17 16 10
- this is a slight compromise to keep some of these more presentational components to not destructure the crazy fusion props 

todo: 

- [x] Test with multiple primary-font switching based off of different sites https://github.com/WPMedia/Fusion-News-Theme/blob/master/blocks.json#L158 vs https://github.com/WPMedia/Fusion-News-Theme/blob/master/blocks.json#L194
- [x] Get lighthouse score and/or unused css for before/after this pr -> todo: check after https://corecomponents.arcpublishing.com/pf/all-blocks/?_website=washpost
- [x] Perhaps find a couple other places in the blocks that this could be used to really scale this

### before 

- key part is css in the html doc, and notably total bytes, rather than unused (they're used but just not necessarily different, duplicated)
Screen Shot 2021-02-04 at 13 59 29

- reduce the amount of style calculations 
- https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations?utm_source=lighthouse&utm_medium=devtools 
- 
Screen Shot 2021-02-04 at 14 06 05
Screen Shot 2021-02-04 at 14 03 48

### after

Screen Shot 2021-02-04 at 15 03 21

### CODE REVIEW

File: blocks/headline-block/index.story.jsx [hunk 1]
```diff
@@ -10,9 +10,9 @@ export default {
 export const customHeadline = () => {
   const headlineString = text('headlineString', 'Man Bites Dog');
   // if we want to allow custom fonts https://stackoverflow.com/a/63128475/7491536
-  const primaryFont = text('primaryFont (web-safe)', 'Arial');
+  // const primaryFont = text('primaryFont (web-safe)', 'Arial');
```
Comment (JackHowa) [inline blocks/headline-block/index.story.jsx]: could probably delete this commented-out line. it's a fun demo though to be able to change the font live

File: blocks/shared-styles/package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
   "license": "CC-BY-NC-ND-4.0",
   "main": "index.js",
   "files": [
-    "scss"
+    "scss",
+    "_children"
```
Comment (JackHowa) [inline blocks/shared-styles/package.json:11]: gotta make sure it gets published 😉

File: .storybook/alias/themes.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 const themes = () => ({
-	primaryFont: ''
+  'primary-font-family': 'Arial',
+  'primary-color': 'rgb(255, 105, 180)',
+  'secondary-color': 'rgb(0, 0, 0)'
```
Comment (matthewroach) [inline .storybook/alias/themes.js:4]: These are the same named keys as Fusion provides to production outputs

**Review by JackHowa — COMMENTED** [review]
Looks good to me. LGTM!

**Comment by JackHowa** [comment]
- validated test instructions
Screen Shot 2021-02-04 at 14 13 14

still more opportunity to use this 

Screen Shot 2021-02-04 at 14 15 02

@matthewroach great work!

**Comment by JackHowa** [comment]
verified second part of this with different site as google fonts working as expected 
Screen Shot 2021-02-04 at 14 20 23

**Pull request merged**